docs/source/json_parser.py

```diff
@@ -0,0 +1,42 @@
+import json
+from pathlib import Path
+
+SOURCE_DIR = Path(__file__).parent
+info_json_file_path = SOURCE_DIR / "_tmp/" / "info.json"
+
+# To get all data needed for the release notes of one release,
+# we need to call `get_installer_information` for every _info.json
+# inside the release directory
+
+def get_installer_info(info_json_file_path: Path) -> dict:
+    """
+    Process _info.json file output by constructor to get installer information,
+    including the list of delivered packages
+    """
+    info_json_raw = json.loads(info_json_file_path.read_text())
+    info_dict = {key: info_json_raw[key] for key in info_json_raw.keys()}
```

Comment:
```suggestion
    info_dict = json.loads(info_json_file_path.read_text())
```

The second line is just copying via list comprehension and thus unnecessary